website/docs/r/mssql_database_extended_auditing_policy.html.markdown

```diff
@@ -10,7 +10,7 @@ description: |-
 
 Manages a Ms Sql Database Extended Auditing Policy.
 
-~> **NOTE:** The Database Extended Auditing Policy Can be set inline here as well as with the [mssql_database_extended_auditing_policy resource](mssql_database_extended_auditing_policy.html) resource. You can only use one or the other and using both will cause a conflict.
+~> **NOTE:** The Database Extended Auditing Policy can be set inline here as well as with the [extended_auditing_policy](extended_auditing_policy.html) block of the azurerm_mssql_database resource. You can only use one or the other and using both will cause a conflict.
```

Comment:
After closer inspection of the issue I think this should actually be rephrased to:
```suggestion
~> **NOTE:** The Database Extended Auditing Policy can also be set in the `extended_auditing_policy` block in the [azurerm_mssql_database](mssql_database.html) resource. You can only use one or the other and using both will cause a conflict.
```